tests/kernel/mellanox_config.pm

```diff
@@ -47,27 +49,28 @@ sub run {
     assert_script_run("./" . $mft_version . "/install.sh");
     assert_script_run("mst start");
 
-    # List network devices
-    assert_script_run("lspci|egrep -i 'network|ethernet'");
-    if (script_run("lspci|grep -i mellanox") != 0) {
-        die "echo There is no Mellanox card here";
-    }
-    if (script_run("ls " . $device1) != 0) {
-        die "echo The directory $device1 doesn't exist";
-    }
-    if (script_run("ls " . $device2) != 0) {
-        die "echo The directory $device2 doesn't exist";
-    }
+    if (check_var('BACKEND', 'ipmi')) {
+        # List network devices
+        assert_script_run("lspci|egrep -i 'network|ethernet'");
```

Comment:
we can run this command in VM too, right?  probably it does not make too much sense but I am voting to run as much as possible in VM even in such cases

Reply:
ok, will do so